### PR TITLE
feature sort data user

### PR DESCRIPTION
**Pull request by namdhgc**
+ finish sort and pagination using jQuery

### CODE REVIEW

**Comment by namdhgc** [comment]
@jackdiff Please review my pull request

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -42,13 +42,19 @@ class UserController extends Controller
     public function getData()
     {
         $ModelUser  = new ModelUser();
-        $limit      = 10;
+        $limit      = 2;
```
Comment (jackdiff) [inline app/Http/Controllers/UserController.php]: Use constant instead of magic number

File: app/Http/Controllers/UserController.php [hunk 2]
```diff
@@ -284,4 +290,42 @@ public function deleteData()
         return response()->json($results);
     }
 
+    public function SortPagination()
```
Comment (jackdiff) [inline app/Http/Controllers/UserController.php]: Should unify the function name convention

**Review by jackdiff — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/UserController.php (hunk 2 above)
Comment (namdhgc) [reply app/Http/Controllers/UserController.php]: fixed

File: app/Http/Controllers/UserController.php (hunk 1 above)
Comment (namdhgc) [reply app/Http/Controllers/UserController.php]: done

**Comment by namdhgc** [comment]
@jackdiff i fixed follow your comment, please review again

**Review by jackdiff — APPROVED** [review]

**Pull request merged**